api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -68,30 +68,35 @@ export async function sendToIdoxNexus(
 
   // localAuthority is only parsed for audit record, not client-specific
   const localAuthority = req.params.localAuthority;
-  const uniformClient = getUniformClient();
+  const idoxNexusClient = getIdoxNexusClient();
 
   // confirm that this session has not already been successfully submitted before proceeding
   const submittedApp = await checkUniformAuditTable(payload?.sessionId);
-  const isAlreadySubmitted =
+  const _isAlreadySubmitted =
     submittedApp?.submissionStatus === "PENDING" && submittedApp?.canDownload;
-  if (isAlreadySubmitted) {
-    return res.status(200).send({
-      sessionId: payload?.sessionId,
-      idoxSubmissionId: submittedApp?.submissionId,
-      message: `Skipping send, already successfully submitted`,
-    });
-  }
+  // if (isAlreadySubmitted) {
+  //   return res.status(200).send({
+  //     sessionId: payload?.sessionId,
+  //     idoxSubmissionId: submittedApp?.submissionId,
+  //     message: `Skipping send, already successfully submitted`,
+  //   });
+  // }
```

Comment:
During testing it's convenient to be able to easily resubmit the same app - we'll comment this section back in once "live" to avoid duplicate submissions in production environments :+1: